src/main/java/dev/joopie/jambot/repository/base/BaseRepository.java

```diff
@@ -0,0 +1,59 @@
+package dev.joopie.jambot.repository.base;
+
+import dev.joopie.jambot.models.BaseModel;
+import io.ebean.DB;
+
+import javax.xml.bind.ValidationException;
+import java.util.Collection;
+import java.util.List;
+
+public abstract class BaseRepository<T extends BaseModel<?>> {
```

Comment:
And the same goes for the BaseRepository. Just pre-defined methods to save things.